sqlstore/src/it/resources/db/migration/V9010__test_data.sql

```diff
@@ -789,6 +789,11 @@ INSERT INTO `ServiceRecord`(`recordId`, `instrumentId`, `title`, `details`, `ser
 (2,1,'Seq1_Rec2',NULL,'Service Person',NULL,'2016-01-21',NULL,NULL, FALSE, NULL),
 (3,2,'Seq2_Rec1',NULL,'Service Person',NULL,'2016-01-21','2016-01-21 9:32:00',NULL, TRUE, 2);
 
+INSERT INTO `Instrument_ServiceRecord`(`recordId`, `instrumentId`) VALUES
```

Comment:
You'll have to remove the `instrumentId`s from the insert into `ServiceRecord` above too